west.yml

```diff
@@ -33,7 +33,7 @@ manifest:
 
     - name: golioth-zephyr-boards
       path: deps/modules/lib/golioth-boards
-      revision: v1.1.0
+      revision: eb7476a89618dc1d8b0eded51ddc58c2977661fe
```

Comment:
I'll update this once https://github.com/golioth/golioth-zephyr-boards/pull/17 is merged/released.